float64 wheel_angle
float64 gas_pos
float64 brake_pos
int8 gear
float64 vibration
bool car_on

# these are the 
# states the gear shift can be in
int8 UNKNOWN=-1
int8 PARKED=0
int8 REVERSE=1
int8 NEUTRAL=2
int8 DRIVE=3
int8 DRIVE_2=4
